### ISSUE TITLE
González/Antequera data sets

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1023]
Two  tracks of nucleosome positioning .
paired-end (PE) sequencing. higher resolution than existing  single-end (SE)

Also provided  information to incorporate to the “Conditions” column in the tracks already in the browser.

**Comment by Antonialock** [comment]
we need to load GSM2253905 and GSM2253906

Kim is downloading the data from GEO

I have added the metadata for the tracks

**Comment by ValWood** [comment]
Is that just one file. Maybe the telomeric contig? that's about that size...

**Comment by kimrutherford** [comment]
I've had a look at the other files from GEO for this publication.  They all cover small regions in pombe and cerevisiae.  Do we want to host tracks that have values only in a small region?

I had a quick look at the paper.  I can't see any mention of genome coordinates for the fragments.

**Comment by kimrutherford** [comment]
From Paco:

Dear Val and Kim,                                                                                                                                                                             

I attach the nucleosome wig files for the files SRR3956100 and SRR3956101
corresponding to biological duplicates of nucleosome maps of S. pombe 972
h- made using the paired-end protocol of Illumina as described in González
et al. Genome Res 2016 Nov;26(11):1532-1543. PMID: 27672899

I guess you should be able to upload them directly to the PomBase browser.
If you need any further details, please, let me know.

Best wishes,

Paco
 MN_Spombe_972h-_Rep2.wig
<https://drive.google.com/a/usal.es/file/d/19VN2AARVwOK4ZceMbIb3rnY6Z6UBgC04/view?usp=drive_web>
 MN_Spombe_972h-_Rep1.wig
<https://drive.google.com/a/usal.es/file/d/1TWl42kzPnrFkp7o2Lw4YJGVriTRA_u8a/view?usp=drive_web>

**Comment by ValWood** [comment]
We need to announce with  the explanation from Paco about why these datasets are improved from the existing ones.

**Comment by ValWood** [comment]
"The reason is that they were
made using paired-end (PE) sequencing and the resulting maps are of higher
resolution than those made with single-end (SE) that are in the browser."

**Comment by ValWood** [comment]
@Antonialock do you want to announce?

**Comment by Antonialock** [comment]
Yes, just one issue. 

Only the features and DNA tracks are part of the URL...I'm sure any selected tracks also used to be part of the URL?
@kimrutherford

**Comment by kimrutherford** [comment]
> Only the features and DNA tracks are part of the URL...I'm sure any selected tracks also used to be part of the URL?

After a bit of testing I think the problem was having commas in the track titles.  Most tracks didn't "stick" when I went back to the same URL, but these two tracks did stick:

 - nucleosome positioning in pcr1delta during vegetative growth - Soriano et al. (2013)
 - Poly(A)+ transcripts during vegetative growth (reverse strand) - Lemay et al. (2014)

So I've changed the commas to semicolons and it now seems to work.
